generator/README.md

```diff
@@ -5,6 +5,16 @@ different Special Interest Groups (SIGs) of Kubernetes. The authoritative
 source for SIG information is the `sigs.yaml` file in the project root. All
 updates must be done there.
 
+The schema for this file should be self explanatory. However, if you need to see all the options check out the generator code in `app.go`.
+
+**Time Zone gotcha**:
+Time zones make everything complicated.
+And Daylight Savings time makes it even more complicated.
+Meetings are specified with a time zone and we generate a link to http://www.thetimezoneconverter.com/ so that people can easily convert it to their local time zone.
+To make this work you need to specify the time zone in a way that that web site recognizes.
+Practically, that means US pacific time must be `PT (Pacific Time)`.
```

Comment:
The converter also understands UTC offsets (e.g. `UTC-7`). Perhaps we could use that instead?

Reply:
NVM.. we run into the same issue there then of DST.